x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManager.java

```diff
@@ -214,7 +216,7 @@ public void stopDeployment(TrainedModelDeploymentTask task) {
             processContext = processContextByAllocation.get(task.getId());
         }
         if (processContext != null) {
-            logger.info("[{}] Stopping deployment", task.getModelId());
+            logger.debug("[{}] Stopping deployment", task.getModelId());
```

Comment:
the corresponding `start deployment` message is debug causing the odd situation where stopping was logged but starting not

Reply:
I actually think it'd be useful to have both starting/stopping info log messages. These shouldn't be frequent events and it'd help to have them available in the logs without having to turn debug level on. What do you think?

Reply:
I made both INFO